src/trade_ui.cpp

```diff
@@ -112,6 +113,16 @@ trade_ui::trade_ui( party_t &you, npc &trader, currency_t cost, std::string titl
 {
     _panes[_you]->add_character_items( you );
     _panes[_you]->add_nearby_items( 1 );
+    map &here = get_map();
+    for( const wrapped_vehicle &wv : here.get_vehicles() ) {
+        if( wv.v->is_owned_by( you ) ) {
+            for( const tripoint pos : wv.v->get_points() ) {
+                if( here.inbounds( pos ) && square_dist( you.pos(), pos ) > 1 ) {
+                    _panes[_you]->add_vehicle_items( pos );
```

Comment:
If I use a vehicle scoop to pick up someone else's items, will this allow me to sell these items back to the person?

(Also, does scooping up own items count as stealing right now?)